### PR TITLE
Updated logging calls to allow for LogLevel.

### PR DESCRIPTION
**Pull request by gusemery**
This was not implemented with the last version, and is needed for more complete logging.

Addition of LogLevel to model allows mapping to log_level within the LM API to map the log level within LM Inspector.

### CODE REVIEW

File: LogicMonitor.DataSDK/Api/Logs.cs [hunk 1]
```diff
@@ -123,7 +123,13 @@ public override void _mergeRequest()
 
         }
 
-        
-        
+        private string BuildBody(LogsV1 log)
```
Comment (AakashKhopade) [inline LogicMonitor.DataSDK/Api/Logs.cs:126]: Why is this method required ? This is not used anywhere.

File: LogicMonitor.DataSDK/Model/LogsV1.cs [hunk 2]
```diff
@@ -23,15 +24,18 @@ public LogsV1()
         {
         }
 
-        public LogsV1(string message, Dictionary<string, string> resourceIds, string timeStamp = default, Dictionary<string, string> metaData = default)
+        public LogsV1(string message, Dictionary<string, string> resourceIds, string timeStamp = default, Dictionary<string, string> metaData = default, LogLevel loglevel=default)
```
Comment (AakashKhopade) [inline LogicMonitor.DataSDK/Model/LogsV1.cs:27]: Setting loglevel to default assign "Trace" as loglevel . Whenever loglevel is not assigned it is expected to be empty string.

Comment (gusemery) [reply LogicMonitor.DataSDK/Model/LogsV1.cs:27]: default end up being an empty string; this has nothing to do with Traces.

File: LogicMonitor.DataSDK/Api/Logs.cs (hunk 1 above)
Comment (gusemery) [reply LogicMonitor.DataSDK/Api/Logs.cs:126]: Oops, that isn't even called.. Please remove.

**Pull request closed without merging**